tox.ini

```diff
@@ -13,7 +13,10 @@ install_command =
 deps =
   -r{toxinidir}/requirements.txt
   -r{toxinidir}/test-requirements.txt
-commands = stestr run {posargs}
+  mypy
+commands =
+  stestr run {posargs}
+  mypy --config-file mypy.ini git_machete/cmd.py
```

Comment:
Hmmm maybe `mypy` before `stestr`? usually you first want the feedback on failed typing, and only then if typing is correct &mdash; on failed tests